### PR TITLE
[RFR] These modules are not done

### PR DESCRIPTION
**Pull request by StefanRijnhart**
The status of these modules is still severely lacking (and it was never claimed they were done).

### CODE REVIEW

**Comment by pedrobaeza** [comment]
It's good to have the diferentiation.

:+1:

File: .travis.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ script:
     - pip install -q git+https://github.com/OCA/openupgradelib.git@master
     - createdb $DB
     - wget -q -O- https://github.com/OCA/OpenUpgrade/releases/download/databases/8.0.psql | pg_restore -d $DB --no-owner
-    - MODULES=base,$(sed -n '/^+========/,$p'  openerp/openupgrade/doc/source/modules80-90.rst | grep Done | sed -n '/^|/ s/^|\([0-9a-z_]*\) *|.*$/\1/g p' | paste -d, -s)
+    - MODULES=base,$(sed -n '/^+========/,$p'  openerp/openupgrade/doc/source/modules80-90.rst | grep "Done\|Partial" | sed -n '/^|/ s/^|\([0-9a-z_]*\) *|.*$/\1/g p' | paste -d, -s)
```
Comment (hbrunn) [inline .travis.yml:44]: this wants to be `egrep` or `grep -E`

Comment (StefanRijnhart) [inline .travis.yml:44]: It seems to be working fine though https://travis-ci.org/OCA/OpenUpgrade/builds/138224440#L307

Comment (hbrunn) [inline .travis.yml:44]: yes, the escaped backslash does the trick. Sorry for the noise.

**Pull request merged**